src/megaapi_impl.cpp

```diff
@@ -7372,7 +7439,8 @@ void MegaApiImpl::openfilelink_result(handle ph, const byte* key, m_off_t size,
 	else
 	{
         request->setPublicNode(new MegaNodePrivate(fileName.c_str(), FILENODE, size, 0, mtime, ph, &keystring, a,
-                                                   fingerprint.size() ? fingerprint.c_str() : NULL));
+                                                   fingerprint.size() ? fingerprint.c_str() : NULL,
+                                                   INVALID_HANDLE));
```

Comment:
`INVALID_HANDLE` is the default value of this constructor. Specifying it here is redundant.